### ISSUE TITLE
importing from sqlite db 

### ISSUE THREAD

**Issue by dkebler** [sissbruecker/linkding#932]
I use another bookmarks manager buku and it uses an sqlite db with the following structure below, with example records
https://github.com/jarun/buku

Is there a way I can import this?  Being it's an sqlite databse I can probably export it into other formats  (NDJSON?)

I don't see any documentation that indicates how to import other data.

![image](https://github.com/user-attachments/assets/e41b0e5d-07bf-4a82-b7f8-3e01a5b1c763)

![image](https://github.com/user-attachments/assets/9c22d5c2-bb39-40f4-b1fc-b840e4696d9e)

Also suggestion,  enable "discussions"  for this repo so us users can ask questions without having to  create an issue :).

**Comment by sissbruecker** [comment]
No, those are not compatible. Buku seems to support exporting to the Netscape HTML bookmark format, which you can import into linkding.
